src/components/OutputForm.tsx

```diff
@@ -47,17 +48,43 @@ export const OutputText = ({ label, value }: { label: string, value?: string })
   );
 }
 
-export const OutputTextArea = ({ label, value }: { label: string, value?: string }) => {
+export const OutputTextArea = ({ label, value, rows }: { label: string, value?: string, rows?: number }) => {
+
+  const content = (value !== undefined) &&  (
+    value.split('\n').map((v,i) => {
+      if (v === '') {
+        return (<Typography key={i}><br /></Typography>)
+      }
+      return <Typography key={i} variant="body1">{v}</Typography>
+    })
+  )
+
+  const [collapse, setCollapse] = useState<boolean>(!!rows);
+  const handleClick = () => {
+    if (!rows) { return; }
+    setCollapse(!collapse);
+  }
+
+  let linesToShow = collapse && (Math.max(rows ?? 0, 0) > 0) ? rows : undefined;
```

Comment:
What's the point of the Math.max here? Would this be the same as 
collapse && ((rows ?? 0) > 0) ? ...

Reply:
I don't want negative values.